code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -213,7 +213,7 @@
 		changed_name = custom_name
 	if(changed_name == "" && client)
 		changed_name = client.prefs.custom_names["cyborg"]
-	if(changed_name == "")
+	if(!changed_name || changed_name == "")
```

Comment:
Isn't this invalidated by the L214 anyway? I don't recall if the config name for borg would be blank by default.